modules/log6x-loki-rbac-rules-perms.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
```

Comment:
missing metadata here, should be logging_alerts/custom-logging-alerts

Reply:
None of these modules are going anywhere except the logging 6.0 subfolder. :)